.gitleaksignore

```diff
@@ -1,4 +1,6 @@
 # SEE: https://github.com/gitleaks/gitleaks/blob/master/README.md#gitleaksignore
 
 cd9c0efec38c5d63053dd865e5d4e207c0760d91:docs/guides/Perform_static_analysis.md:generic-api-key:37
+87312c6a627a7b0420956d49187fd15b130df170:src/__tests__/components/molecules/
```

Comment:
you shouldn't need line 4 as this exception is already covered by line 5

Reply:
line 4 removed